common/logger/src/diem_logger.rs

```diff
@@ -239,7 +240,13 @@ impl DiemLoggerBuilder {
                 let mut filter_builder = Filter::builder();
 
                 if self.is_async && self.address.is_some() {
-                    filter_builder.filter_level(self.remote_level.into());
+                    if env::var(RUST_LOG_REMOTE).is_ok() {
+                        filter_builder.with_env(RUST_LOG_REMOTE);
+                    } else if env::var(RUST_LOG).is_ok() {
+                        filter_builder.with_env(RUST_LOG);
```

Comment:
looking at this code it feels like RUST_LOG and RUST_LOG_REMOTE have no difference in behavior?

Reply:
This code does the following in order to set the filter configuration for what logs should be sent to the remote logging service:
If RUST_LOG_REMOTE is set use it, else if RUST_LOG is set use it (as a sane fallback path since most folks are familiar with the RUST_LOG environment variable), else just use the default Info level.

One question I have (I don't remember the original rationale) is do we want/need to support configuring the logs that are sent to the pod vs what are sent to the central logging service. If anything having two different configurations seems to add more confusion. Maybe it would be good to unify this and only use one filter to determine what logs should be captured.

Reply:
gah missed the two filters